test/inductor/test_binary_folding.py

```diff
@@ -159,6 +160,9 @@ def my_inner_compile(gm, example_inputs, *args, **kwargs):
             )
 
     def test_conv_bn_folding(self):
```

Comment:
May be use

```
@inductor_config.patch({"efficient_conv_bn_eval_fx_passes ": False})
def test_conv_bn_folding(self):
```

Reply:
I cannot find the `inductor_config.patch` function. Does this functionality exist? Patching test would be great.

Reply:
see this test case: https://github.com/pytorch/pytorch/blob/main/test/inductor/test_torchinductor.py#L6242

Reply:
Thanks! Fixed in [a2444a6](https://github.com/pytorch/pytorch/pull/108757/commits/a2444a60250c4344dcea41dff9d55c5da41fd464).

Reply:
@eellison is [5046dea](https://github.com/pytorch/pytorch/pull/108757/commits/5046deaf2587a42e9e6112a4037454e98c0897db) what you want? I add an additional check for the new pass: disable it if freezing is on.